### PR TITLE
feat: my kiva navigation

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- my kiva navigation added

Screenshot 2024-09-19 at 5 31 20 p m

### CODE REVIEW

File: src/pages/Portfolio/MyKiva/MyKivaPage.vue [hunk 1]
```diff
@@ -18,22 +26,43 @@ export default {
...
+			this.userInfo = result.data?.my ?? {};
 			this.lender = result.data?.my?.lender ?? null;
 		},
 	},
+	computed: {
+		userBalance() {
+			return this.userInfo?.userAccount?.balance ?? '';
+		},
+		lenderName() {
+			return this?.lender?.name ?? '';
```
Comment (dyersituations) [inline src/pages/Portfolio/MyKiva/MyKivaPage.vue:54]: this? optional chaining isn't needed for this or `lenderImageUrl`

File: src/pages/Portfolio/MyKiva/MyKivaPage.vue [hunk 2]
```diff
@@ -18,22 +26,43 @@ export default {
...
+		},
+		lenderImageUrl() {
+			return this?.lender?.image?.url ?? '';
+		},
+	},
+	methods: {
+		handleShowNavigation() {
+			this.showNavigation = true;
+			this.$kvTrackEvent('SecondaryNav top level', 'click', 'MyKiva-Settings-icon');
```
Comment (dyersituations) [inline src/pages/Portfolio/MyKiva/MyKivaPage.vue:63]: We need to use pre-approved categories for events, so this should probably just have a category of `borrower-profile`

File: src/components/MyKiva/MyKivaNavigation.vue [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+							v-for="setting in profileSettingsOptions"
+							:key="setting.text"
+							class="tw-mt-1.5
+								hover:tw-translate-x-1 hover:lg:tw-translate-x-2 tw-duration-300 tw-delay-100"
+						>
+							<router-link
+								class="tw-text-primary tw-font-medium"
+								:to="setting.link"
+								v-kv-track-event="[
+									'SecondaryNav links',
```
Comment (dyersituations) [inline src/components/MyKiva/MyKivaNavigation.vue:60]: This should probably be a category of `portfolio`

File: src/components/MyKiva/MyKivaNavigation.vue [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+							:key="setting.text"
+							class="tw-mt-1.5
+								hover:tw-translate-x-1 hover:lg:tw-translate-x-2 tw-duration-300 tw-delay-100"
+						>
+							<component
+								:is="setting.isDonate && !parseFloat(userBalance) ? 'span' : 'router-link'"
+								class="tw-text-primary tw-font-medium"
+								:to="returnLendingUrl(setting)"
+								v-kv-track-event="[
+									'SecondaryNav links',
```
Comment (dyersituations) [inline src/components/MyKiva/MyKivaNavigation.vue:86]: Same.

File: src/components/MyKiva/MyKivaNavigation.vue [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+							v-for="setting in communityOptions"
+							:key="setting.text"
+							class="tw-mt-1.5
+								hover:tw-translate-x-1 hover:lg:tw-translate-x-2 tw-duration-300 tw-delay-100"
+						>
+							<router-link
+								class="tw-text-primary tw-font-medium"
+								:to="setting.link"
+								v-kv-track-event="[
+									'SecondaryNav links',
```
Comment (dyersituations) [inline src/components/MyKiva/MyKivaNavigation.vue:111]: Same.

File: src/pages/Portfolio/MyKiva/MyKivaPage.vue (hunk 2 above)
Comment (roger-in-kiva) [reply src/pages/Portfolio/MyKiva/MyKivaPage.vue:63]: Agreed, but this category is the one mentioned in the ticket: https://kiva.atlassian.net/browse/MP-729
cc @emuvente

Comment (emuvente) [reply src/pages/Portfolio/MyKiva/MyKivaPage.vue:63]: Yeah, just follow what's in the ticket and don't worry about whether the event details are following the web events guide. It does not seem to be something that product or analytics care about strongly any more.

**Review by emuvente — APPROVED** [review]

File: src/pages/Portfolio/MyKiva/MyKivaPage.vue (hunk 2 above)
Comment (dyersituations) [reply src/pages/Portfolio/MyKiva/MyKivaPage.vue:63]: @emuvente I had thought we needed to only use validated strings, but I see that this list doesn't include categories that we are currently using, for example `Lending`, so it must not be a requirement.

https://github.com/kiva/analytics-notebooks/blob/446439ec4993aaf5f8e05c18dc7e674d8e72b509/notebooks/analytics/structured_event_validation/se_validation.json

**Review by dyersituations — APPROVED** [review]

File: src/pages/Portfolio/MyKiva/MyKivaPage.vue (hunk 2 above)
Comment (mcstover) [reply src/pages/Portfolio/MyKiva/MyKivaPage.vue:63]: May want to check with Greg to be sure we can import these new event types into Amplitude if that is expected. I think that list is related to the automated export/import into that tool.

Comment (dyersituations) [reply src/pages/Portfolio/MyKiva/MyKivaPage.vue:63]: Ah, so Looker will work with anything, but potentially not Amplitude.

**Pull request merged**